bip-MarcoPon-BlockchainURI.mediawiki

```diff
@@ -0,0 +1,80 @@
+<pre>
+  BIP: bip-MarcoPon-BlockchainURI
+  Title: URI scheme for Blockchain references / exploration
+  Author: Marco Pontello <marcopon@gmail.com>
+  Status: Draft
+  Type: Standards Track
+  Created: 29 August 2015
```

Comment:
Can you add the following?

Post-History: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-August/010712.html